lib/metadata/schema-parser.js

```diff
@@ -941,6 +1004,79 @@ SchemaParserV2.prototype._parseUdt = function (udtInfo, row, callback) {
   });
 };
 
+/**
+ * Used to parse schema information for Cassandra versions 4.x and above.
+ *
+ * This parser similar to [SchemaParserV2] expect it also parses virtual
+ * keyspaces.
+ *
+ * @param {ControlConnection} cc The control connection to be used
+ * @param {Function} udtResolver The function to be used to retrieve the udts.
+ * @constructor
+ * @ignore
+ */
+function SchemaParserV3(cc, udtResolver) {
+  SchemaParserV2.call(this, cc, udtResolver);
+  this.selectVirtualTable = _selectVirtualTable;
```

Comment:
NIT: We can use the `const` directly without the need of a instance level field.

Reply:
:+1: , made that change, but also introduced a `supportsVirtual` instance field, as I still needed a way of communicating that a parser supported virtual tables (might be a better way though).